templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -52,7 +52,11 @@ jobs:
           git checkout {{ plugin_default_branch }}
           bump2version --no-commit minor
 
-      - name: Make a PR with version bump
+      - name: Remove entries from CHANGES directory
+        run: |
+          find CHANGES -type f -regex ".*\.\(bugfix\|doc\|feature\|misc\)" -exec git rm {} +
```

Comment:
There's two more: removal & deprecation. https://github.com/pulp/pulpcore/blob/main/pyproject.toml#L20-L50

Reply:
Great eye! I added both!

Reply:
Actually, there is a default set or it is configured in pyproject.yaml. How much dynamic do we want to make it?